alias/gui.c

```diff
@@ -126,6 +136,22 @@ static struct AliasView *alias_view_new(void)
   return mutt_mem_calloc(1, sizeof(struct AliasView));
 }
 
+/**
+ * alias_get_sort_function - Sorting function decision logic
+ */
+int (*alias_get_sort_function())(const void *a, const void *b)
+{
+  switch ((C_SortAlias & SORT_MASK))
+  {
+    case SORT_ALIAS:
+      return alias_sort_name;
+    case SORT_ADDRESS:
+      return alias_sort_address;
+    case SORT_UNSORT:
+      return alias_sort_unsort;
```

Comment:
It shouldn't be possible for any other values, but...
The default for `$sort_alias` is "alias", so I'd move `SORT_ALIAS` to the end and add a `default:`

Reply:
I simply added a default, I think it is fine now? Didn't understand what you meant by moving `SORT_ALIAS` to the end.

Reply:
That's fine.
I simply meant moving the `case SORT_ALIAS` to the end of the `switch`, so that it could be combined with the `default`.